### PR TITLE
fix(Tooltip): update position if changed beyond threshold

### PR DESCRIPTION
**Pull request by Noodles006**
Applies to any of the following positions: top/right/bottom/left/width/height

### CODE REVIEW

**Comment by Noodles006** [comment]
@chrisdhanaraj Could you help review this updates for Tooltip component. Thanks.

**Comment by hellobrian** [comment]
For future reference: [commit message guidelines](https://github.com/carbon-design-system/carbon-components-react/blob/master/CONTRIBUTING.md#commit-message-guidelines)

File: components/Tooltip.js [hunk 1]
```diff
@@ -22,23 +23,36 @@ class Tooltip extends Component {
     showIcon: true,
     iconName: 'info--glyph',
     iconDescription: 'tooltip',
+    positionThreshold: 5,
```
Comment (chrisdhanaraj) [inline components/Tooltip.js]: @Noodles006 apologies was out last week.

For clarification - why 5 / why a prop at all? Seems to me the default behavior should be that if it exceeds a certain distance away to always recalculate the position

Comment (Noodles006) [inline components/Tooltip.js]: @chrisdhanaraj there is a property named triggerPosition in the state, which indicates the position of the tooltip content, right? but this property will only been calculate when component did mounted, so if a tooltip component in dialog and the component's position is updated, then the tooltip content will not been updated.

**Comment by iangfleming** [comment]
Hey! Can you go into further detail about why the threshold prop is needed? I have a similar fix but it doesn't have a threshold: https://github.com/carbon-design-system/carbon-components-react/pull/81
I feel like I'm not quite understanding your fix and I just wanted to make sure I'm not missing something important 🙂

**Comment by chrisdhanaraj** [comment]
Closing for lack of response.

**Pull request closed without merging**